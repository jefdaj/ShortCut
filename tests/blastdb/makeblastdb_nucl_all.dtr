small6803fna = load_fna "data/6803genes10.fna"
large6803fna = load_fna "data/6803genes.fna"
db = makeblastdb_nucl_all [small6803fna, large6803fna]
result = db
