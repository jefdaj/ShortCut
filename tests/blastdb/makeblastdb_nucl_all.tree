.
|-- cache
|   |-- load
|   |   |-- 6ea57f1208.fna
|   |   |-- 6ea57f1208.fna.ids
|   |   |-- 737dfdb34e.fna
|   |   `-- 737dfdb34e.fna.ids
|   `-- makeblastdb
|       `-- 3ce4648d84
|           |-- 3ce4648d84.ndb.nhr
|           |-- 3ce4648d84.ndb.nin
|           `-- 3ce4648d84.ndb.nsq
|-- exprs
|   |-- list
|   |   `-- 18a2e339c8.fna.list -> ../../cache/lines/...
|   |-- load_fna
|   |   |-- 34a32d8e2f.fna -> ../../cache/load/737dfdb34e.fna
|   |   `-- 45126bfa2b.fna -> ../../cache/load/6ea57f1208.fna
|   |-- makeblastdb_nucl_all
|   |   `-- 2bb60fcdd8.ndb -> ../../cache/lines/...
|   `-- str
|       |-- 7da80bde02.str -> ../../cache/lines/...
|       `-- a2fd369878.str -> ../../cache/lines/...
`-- vars
    |-- db.ndb -> ../exprs/makeblastdb_nucl_all/2bb60fcdd8.ndb
    |-- large6803fna.fna -> ../exprs/load_fna/34a32d8e2f.fna
    |-- result -> ../vars/db.ndb
    `-- small6803fna.fna -> ../exprs/load_fna/45126bfa2b.fna

10 directories, 17 files
