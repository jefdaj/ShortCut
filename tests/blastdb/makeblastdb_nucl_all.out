Database: 3ce4648d84.ndb
3,574 sequences; 3,432,498 total bases
Longest sequence: 12,600 bases
Volumes:
$TMPDIR/cache/makeblastdb/3ce4648d84/3ce4648d84.ndb

